test_scripts/collect_data.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Add a comment section here to describe what this script does.
A good comment should have: 1) the name of the script; 2) the purpose of the script; 3) a brief on how it was implemented (1-2 sentences)